perun-base/src/main/resources/perun-roles.yml

```diff
@@ -7016,32 +7120,44 @@ perun_policies:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   group-updateFormItem_ApplicationFormItem_policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   copyFormFromVoToVo_Vo_Vo_policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   source-copyFormFromVoToGroup_Vo_Group_Policy:
     policy_roles:
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
 
   destination-copyFormFromVoToGroup_Vo_Group_Policy:
     policy_roles:
       - VOADMIN: Vo
       - GROUPADMIN: Group
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Vo
+      - MFA: Group
 
   source-copyFormFromGroupToGroup_Group_Group_policy:
```

Comment:
I'm not sure if you skipped it or if it's correct, but shouldn't there be mfa rules like for  `destination-copyFormFromGroupToGroup_Group_Group_policy`? The rest looks fine to me.

Reply:
I thought the source group won't be changed, only its form would be copied so it is a read operation = non critical. What do you think?